src/collision.c

```diff
@@ -0,0 +1,1084 @@
+#include "common.h"
+#include "model.h"
+
+extern s16 D_800D91DC;
+
+typedef struct {
+    u32 collisionOffset;
+    u32 zoneOffset;
+} HitAsset;
```

Comment:
Having offset / size comments for these would be nice, as we do for other structs (see common_structs.h)